client/src/main/resources/templates/js/main.js

```diff
@@ -431,31 +431,31 @@ function displaySource(src, title) {
         configTotal = configFilterSearch.length;
       }
 
-      $(pageAdditionalId + 'testcase-pagination ul .page-number').remove();
+      $(pageAdditionalId + "testcase-pagination ul .page-number").remove();
       renderPageNumbers(pageAdditionalId, 1, getNoOfPage(pageAdditionalId));
       renderTable(pageAdditionalId, getFilterSearch(pageAdditionalId), 1, getPageSize(pageAdditionalId));
     });
   }
 
   function renderPagination(pageAdditionalId) {
-    $(pageAdditionalId + 'testcase-pagination ul .page-first').click(function() {
+    $(pageAdditionalId + "testcase-pagination ul .page-first").click(function() {
       if(1 !== getIndex(pageAdditionalId)) {
         setIndex(pageAdditionalId, 1);
-        $(pageAdditionalId + 'testcase-pagination ul .page-number').remove();
+        $(pageAdditionalId + "testcase-pagination ul .page-number").remove();
         renderPageNumbers(pageAdditionalId, 1, getNoOfPage(pageAdditionalId));
         renderTable(pageAdditionalId, getFilterSearch(pageAdditionalId), getIndex(pageAdditionalId),
           getPageSize(pageAdditionalId));
       }
     });
 
-    $(pageAdditionalId + 'testcase-pagination ul .page-pre').click(function() {
+    $(pageAdditionalId + "testcase-pagination ul .page-pre").click(function() {
       if(0 < (getIndex(pageAdditionalId) - getPageSize(pageAdditionalId))) {
         setIndex(pageAdditionalId, (getIndex(pageAdditionalId) - getPageSize(pageAdditionalId)));
-        var k = $(pageAdditionalId + 'testcase-pagination ul .page-number').first().find('a').text();
+        var k = $(pageAdditionalId + "testcase-pagination ul .page-number").first().find("a").text();
         var n = parseInt(k) - 1;
         if(n > 0) {
-          $(pageAdditionalId + 'testcase-pagination ul .page-number').last().remove();
-          $(pageAdditionalId + "testcase-pagination ul .page-pre").after($('<li>').attr('data-index',n).addClass('page-number').append($('<a>').attr('href','#').append(n) ));
+          $(pageAdditionalId + "testcase-pagination ul .page-number").last().remove();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l458)